rest/utilities_testing.go

```diff
@@ -689,6 +709,15 @@ func (bt BlipTester) Close() {
 	bt.restTester.Close()
 }
 
+// Returns database context for blipTester (assumes underlying rest tester is based on a single db - returns first it finds)
+func (bt BlipTester) DatabaseContext() *db.DatabaseContext {
```

Comment:
We're chaining `bt.DatabaseContext().NewDCPCachingCountWaiter(...)` in a few places, which would panic if this doesn't find a context.

We should either return an error from this function as well, to prevent chaining, or handle a nil pointer receiver in `NewDCPCachingCountWaiter`